cmd/minishift/cmd/hostfolder/add_test.go

```diff
@@ -42,6 +42,21 @@ func Test_host_folder_name_required(t *testing.T) {
 	addHostFolder(nil, nil)
 }
 
+func Test_host_folder_name_not_empty(t *testing.T) {
```

Comment:
@anjannath Some set of lines are common in all the Test_ function. Can we keep it separate function and make a call when it is needed ?
```
var err error
tmpMinishiftHomeDir := cli.SetupTmpMinishiftHome(t)	config.InstanceConfig, err = config.NewInstanceConfig(filepath.Join(tmpMinishiftHomeDir, "config"))
config.AllInstancesConfig, err = config.NewAllInstancesConfig(filepath.Join(tmpMinishiftHomeDir, "config"))	assert.NoError(t, err, "Unexpected error setting instance config")

tee := cli.CreateTee(t, true)
defer cli.TearDown(tmpMinishiftHomeDir, tee)
defer viper.Reset()
```

Reply:
@amitkrout I Think this should be done as part of a refactor, a separate commit. WDYT? This PR is only for the fix about blank names.

Reply:
Or i can add a separate commit for removing the duplications

Reply:
> I Think this should be done as part of a refactor

+1, lets keep it in a separate pr. Please create another issue something like ```refactor unit test to remove duplicate lines```

Reply:
> Please create another issue something like refactor unit test code to remove duplicate lines

+1